app/views/sessions/home.html.erb

```diff
@@ -1,24 +1,33 @@
 <style>
     body {
         flex: 1 1 auto;
+        text-align: center;
     }
 
     div.aRoute {
+        position:relative; 
+        left:80px; 
+        top:80px;
         background-color: #eeeeee;
```

Comment:
switched from absolute to relative as absolute would hold all the cards in the same place